lib/components/primitive-components/Group/Group_doInitialPcbLayoutPack/applyPackOutput.ts

```diff
@@ -87,6 +114,7 @@ export const applyPackOutput = (
             "pcb_component_id" in elm && elm.pcb_component_id === componentId,
         )
       transformPCBElements(related as any, transformMatrix)
+      updateCadRotation(db, componentId, rotationDegrees, pcbComponent.layer)
```

Comment:
1. named parameters when there are more than 2 args
2. fully qualified id variable names _always_, `componentId` should be `pcbComponentId`
3. Don't change the name of the variables throughout the code, i see `memberId` in some places as well!